packages/console/src/components/Checkbox/index.module.scss

```diff
@@ -0,0 +1,45 @@
+@use '@/scss/underscore' as _;
+
+.checkbox {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+
+  .icon {
+    width: 20px;
+    height: 20px;
+    margin-right: _.unit(2);
+    display: none;
+  }
+
+  input {
+    position: absolute;
+    width: 20px;
+    height: 20px;
+    left: 0;
+    top: 0;
+    margin: 0;
+    opacity: 0%;
+  }
+
+  input:not(:checked):not(:disabled) ~ .uncheck {
+    display: block;
+  }
+
+  input:not(:checked):disabled ~ .uncheckDisabled {
+    display: block;
+  }
+
+  input:checked:not(:disabled) ~ .check {
+    display: block;
+  }
+
+  input:checked:disabled ~ .checkDisabled {
+    display: block;
+  }
```

Comment:
nit: we can combine these selectors together, and separate them by comma

Reply:
updated